src/extension/loadLogs.ts

```diff
@@ -11,7 +11,7 @@ import * as Telemetry from './telemetry';
 
 export const driverlessRules = new Map<string, ReportingDescriptor>();
 
-export async function loadLogs(uris: Uri[], token?: { isCancellationRequested: boolean }) {
+export function loadLogs(uris: Uri[], token?: { isCancellationRequested: boolean }): Log[] {
```

Comment:
Prefer to defer housekeeping to another PR. There are other calls to `loadLogs` you have not fixed up consistently. But to fix all of them detracts from the clarity of this PR.